### ISSUE TITLE
feat: AI Application Simulator 頁面 - 主架構與 NES Model

### ISSUE THREAD

**Issue by thc1006** [thc1006/new_frontendv3#15]
## 概述
根據 Figma 設計實作 AI Application Simulator 頁面的主架構與 NES Model 功能。

## Figma 參考
- [Main Menu (277:952)](https://www.figma.com/design/P7bPVphHwCiPDTQGRVcAK3?node-id=277-952)
- [NES Select Initial (277:465)](https://www.figma.com/design/P7bPVphHwCiPDTQGRVcAK3?node-id=277-465)
- [NES Pre-train Running (277:383)](https://www.figma.com/design/P7bPVphHwCiPDTQGRVcAK3?node-id=277-383)
- [NES Pre-train Finish (277:342)](https://www.figma.com/design/P7bPVphHwCiPDTQGRVcAK3?node-id=277-342)
- [NES Preview (277:296)](https://www.figma.com/design/P7bPVphHwCiPDTQGRVcAK3?node-id=277-296)

## 路由
\`/projects/[projectId]/ai-simulator\`

### 主頁面架構
- [ ] 左側 Model List 面板 (NES, Positioning, IM, MRO, RS, BC)
- [ ] 右側 Model Inference 面板
- [ ] Scene 下拉選單
- [ ] 警告提示 "Note: All data is generated by the simulator."

### NES Model 功能
- [ ] Model Select 下拉選單
- [ ] Pre-train 按鈕
- [ ] Preview 按鈕
- [ ] START/STOP 按鈕狀態切換
- [ ] Training Status 顯示 (running/Finish)
- [ ] Training Results 顯示 (Reward, Actor Loss, Critic Loss)
- [ ] Epoch 進度顯示
- [ ] Loss Function 資訊
- [ ] Update 按鈕 (訓練完成後)

### NES Preview 功能
- [ ] Scene 選擇器 (None, 上班, 下班, 上課1/隨機, 上課2/同步)
- [ ] 地圖預覽

## 後端 API (需 placeholder)
```
GET    /projects/{id}/ai-simulator/models
GET    /projects/{id}/ai-simulator/scenes
POST   /projects/{id}/ai-simulator/pretrain/start
POST   /projects/{id}/ai-simulator/pretrain/stop
GET    /projects/{id}/ai-simulator/pretrain/status
POST   /projects/{id}/ai-simulator/pretrain/update
GET    /projects/{id}/ai-simulator/preview
```

## 驗收條件
- [ ] 頁面 UI 對齊 Figma 設計
- [ ] Pre-train 流程狀態切換正常
- [ ] E2E 測試覆蓋
